packages/mergebot/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "dt-mergebot",
+  "version": "3.0.0",
+  "description": "The bot that manages DefinitelyTyped PRs.",
+  "author": "DT Maintainers",
+  "homepage": "https://github.com/DefinitelyTyped/dt-mergebot#readme",
+  "main": "dist/functions/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/DefinitelyTyped/dt-mergebot.git"
+  },
+  "bugs": {
+    "url": "https://github.com/DefinitelyTyped/dt-mergebot/issues"
+  },
+  "license": "MIT",
+  "engines": {
+    "node": ">=18"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.10.3",
+    "@azure/functions": "^4.4.0",
+    "@definitelytyped/old-header-parser": "npm:@definitelytyped/header-parser@0.0.178",
```

Comment:
Once the package-lock is gone and the root pnpm lock is updated, will need to verify that this still hits npm for the old revision.

Reply:
Seems like yes:
```
      '@definitelytyped/old-header-parser':
        specifier: npm:@definitelytyped/header-parser@0.0.178
        version: '@definitelytyped/header-parser@0.0.178'
```